UPGRADE.md

```diff
@@ -7,6 +7,16 @@ not skip minor/major releases while upgrading (fix releases can be skipped).
 The format is inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 3.0.0 to 3.0.1
+
+### Before switching
+
+- AWS lambda-encode must be deployed before switching to this version.
```

Comment:
re-deployed or updated?

Reply:
For each release we made all lambda must be deployed. Here it is important to deploy them before running the migration.